middlewares/validators/tasks.js

```diff
@@ -23,7 +23,7 @@ const createTask = async (req, res, next) => {
       assignee: joi.string().optional(),
       priority: joi.string().required(),
       percentCompleted: joi.number().required(),
-      dependsOn: joi.array().items(joi.string()).optional(),
+      dependsOn: joi.array().items().optional(),
```

Comment:
the array items will be still `string`. why are we removing it here then ?

Reply:
resolved